app/helpers/invite_helper.rb

```diff
@@ -0,0 +1,19 @@
+module InviteHelper
+  def invite_booth
+    if booth_params[:invited_users]
+      emails_array = booth_params[:invited_users].split(',')
+
+      emails_array.each do |email|
+        new_user = User.find_by(email: email)
+        if new_user.nil? || !new_user.confirmed?
```

Comment:
`!new_user.confirmed?` that's not bulletproof. It's possible that we have unconfirmed users (when using) but the users are actually able to log in and don't require invitation.

Reply:
@differentreality we can add user's "last_sign_in_at", if it is nil then we can send invitations again, is that okay?

Reply:
If there is already a user record, why do we need to check anything else?

I mean there are a few ways we can go with this approach and make the feature more useful and meaningful, but what is currently being implemented is pretty basic, so I don't think we should add any more functionality than what is absolutely necessary, unless you plan on documenting (and adding tests) for the more advanced scenarios.